src/partials/resource-limits.md

```diff
@@ -0,0 +1,61 @@
+## Check resource usage {% #check-resource-usage %}
+
+You can check your organization's resource usage by navigating to your organization, under the **Usage** tab, you can check the current billing cycle's usage stats.
+
+
+## Reaching resource limits {% #reaching-resource-limits %}
+Reaching your organization's resource limits will have the following effects until the current billing period ends.
+{% table %}
+* Component
+* Consequence
+---
+* **Bandwidth** {% rowspan=3 %}
+*  More bandwidth will be purchased automatically until organization reaches budget cap. If the organization uses a Starter plan or the budget cap is reached, API access will be denied until your organization's plan is upgraded or your budget cap is increased.
+---
+*  Importing projects via migrations disabled, but you can still export your projects.
+---
+*  Platform creation disabled.
+---
+* **Monthly active users**
```

Comment:
For billing v1 I don't think we can use the term `Monthly active users`. It should be "Users" here. Is that correct @lohanidamodar @ArmanNik ?

Reply:
@heyCarla that's correct

Reply:
Fixed

Reply:
I think you need to push this change? Looks the same to me

Reply:
Ah yes, imagine pushing after making changes. Thanks